frontend/src/components/common/MessageContainerClassroom.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
```

Comment:
Instead of adding a whole new `MessageContainer` component, it would be best to reuse the existing one and just do a quick refactor to add an optional `onClick` prop. See the `HeaderWithButton` component for an idea of how you might do this!